package.json

```diff
@@ -64,13 +64,14 @@
   "dependencies": {
     "babel": "^5.8.19",
     "btoa": "^1.1.2",
-    "fake-indexeddb": "^1.0.3",
+    "fake-indexeddb": "1.0.3",
```

Comment:
This is because latest 1.0.4 introduces a failure with one of its dependencies, which I'm about to file an issue for.